app/handlers/config.py

```diff
@@ -2,21 +2,50 @@
 
 import tornado.web
 
+from trafficcloud.pm import default_config_dict, update_project_config
```

Comment:
Perhaps we should move the pm.py file to traffic)cloud)utils now? It seems like we can remove the old trafficcloud folder if we do so.